src/code42cli/cmds/cases.py

```diff
@@ -151,7 +151,10 @@ def _list(
     formatter = OutputFormatter(format, _get_cases_header())
     cases = [case for page in pages for case in page["cases"]]
     if cases:
-        formatter.echo_formatted_list(cases)
+        if len(cases) > 10:
```

Comment:
is this logic everywhere where paging happens? could we abstract it somehow? For example, if we decided to increase the number of records required in order for paging to occur, how many places would we need to touch?

Reply:
If we ever do the ticket that switched the formatters to always use the `DataFrameFormatter`, it would be resolved. I am not sure of what all that entails though. In the short term, I can put something in

Reply:
I just moved the pager logic to the `echo_formatted_output` anyway since it is always used at the same time